packages/agents/src/utils.ts

```diff
@@ -0,0 +1,16 @@
+export const extractRequires = (code: string) => {
```

Comment:
Having this in `agents/src/utils.ts` feels wrong, let's keep it localized to where it's used (`WriteScript` function)